IBAnimatableApp/TransitionTableViewController.swift

```diff
@@ -185,82 +185,47 @@ fileprivate extension TransitionAnimationType {
   var asString: String {
     switch self {
     case .fade(let direction):
-      return "Fade" + "(\(direction.asString))"
+      return "fade" + "(\(direction.rawValue)"
     case .systemCube(let direction):
-      return "SystemCube" + "(\(direction.asString)"
+      return "systemCube" + "(\(direction.rawValue)"
     case .systemFlip(let direction):
-      return "SystemFlip" + "(\(direction.asString))"
+      return "systemFlip" + "(\(direction.rawValue))"
     case .systemMoveIn(let direction):
-      return "SystemMoveIn" + "(\(direction.asString))"
+      return "systemMoveIn" + "(\(direction.rawValue))"
     case .systemPush(let direction):
-      return "SystemPush" + "(\(direction.asString))"
+      return "systemPush" + "(\(direction.rawValue))"
     case .systemReveal(let direction):
-      return "SystemReveal" + "(\(direction.asString))"
+      return "systemReveal" + "(\(direction.rawValue))"
     case .systemPage(let type):
-      return "SystemPage(\(type.asString))"
+      return "systemPage(\(type.rawValue))"
     case .systemCameraIris(let hollowState):
-      return hollowState == .none ? "SystemCameraIris" : "SystemCameraIris" + "(\(hollowState.asString))"
+      return hollowState == .none ? "systemCameraIris" : "systemCameraIris" + "(\(hollowState.rawValue))"
```

Comment:
Here we can do  `return "systemCameraIris" + hollowState == .none ? "" : "(\(hollowState.rawValue))"` to reduce one duplicated string, but not a big deal 😁